packages/core/core-flows/src/product/workflows/update-products.ts

```diff
@@ -20,7 +33,60 @@ export const updateProductsWorkflow = createWorkflow(
     input: WorkflowData<WorkflowInput>
   ): WorkflowData<ProductTypes.ProductDTO[]> => {
```

Comment:
Let's add at least 1 test in `product.js` that ensures this change is working as expected.